tox.ini

```diff
@@ -0,0 +1,26 @@
+# tox (https://tox.readthedocs.io/) is a tool for running tests
+# in multiple virtualenvs. This configuration file will run the
+# test suite on all supported python versions. To use it, "pip install tox"
+# and then run "tox" from this directory.
+
+[tox]
+# Choose your Python versions. They have to be available
+# on the system the tests are run on.
+envlist = py27, py36
+
+# Tell tox to not require a setup.py file
+skipsdist = True
+
+[testenv]
+whitelist_externals = mkdir
+deps =
+  -rdev-requirements.txt
+  -rrequirements.txt
+
+commands=
+  mkdir -p {toxworkdir}/tests/{envname}/nosetests
```

Comment:
From `tox` documentation 

> toxworkdir=path   # tox working directory, defaults to {toxinidir}/.tox

If you check your build log, you will see that it indeed creates a directory at 

> py27 create: /home/ubuntu/scrapy-datadog-extension/.tox/py27

But it is not visible in the artifact tab in circle-ci. Maybe you need to set these paths properly.